components/dada-lang/src/test_harness.rs

```diff
@@ -166,35 +166,31 @@ impl Options {
             &expected_diagnostics.compile,
             &mut errors,
         )?;
-        self.check_output_against_ref_file(
+        self.maybe_bless_ref_file(
```

Comment:
I was going to suggest that we make this *always* bless, but maybe it's good to have a mode that doesn't generate output files. (I somewhat wonder if `--bless` should be the default, even, and instead we have `--check`, but I guess that could be surprising.)

Reply:
Could be a good idea to always bless by default and ensure you're not looking at old files.

Reply:
If you wanted to open up a follow-up PR, would be welcome. Maybe a good idea would be to *try* and update the ref files, but if an I/O error occurs, just *warn* about it and don't fail the tes.t